core/.gitignore

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
should this be `legacy-design/core/.gitignore`? 
Same question for the other folders not under `legacy-design` or `adr`

Reply:
yep - will add @lenny-intel

Reply:
BTW - all the .gitignores are just to make sure the directories stay in place.  In the revision, I have a top level /design folder with /adr, /legacy-design and /legacy-requirement folders.  The /adr folder has all the workgroup sub folders (core, device-services, etc.)